apis/v1alpha1/forwarding_target_types.go

```diff
@@ -65,6 +65,18 @@ type ForwardToTarget struct {
 	// +optional
 	// +kubebuilder:default=1
 	Weight TargetWeight `json:"weight" protobuf:"bytes,3,opt,name=weight"`
+
+	// Filters defined at this-level should be executed if and only if
+	// the request is being forwarded to the target defined here.
+	//
+	// Conformance: For any implementation, filtering support, including core
+	// filters, is NOT guaranteed at this-level.
+	// Use Filters in HTTPRouteRule for portable filters across implementations.
+	//
+	// Support: custom
+	//
+	// +optional
+	Filters []HTTPRouteFilter `json:"filters" protobuf:"bytes,4,rep,name=filters"`
```

Comment:
I think I may have already asked this, but do we have examples of what filters might be set here instead of HTTPRouteRule? If this is something we're considering as a custom level of support would it make sense to leave this until after v1alpha1? This is an area I'm less familiar with, there likely are very real use cases here, it just seems like it may be difficult for some implementations to support this.

Reply:
#55 is the reason for adding this and multiple folks have asked for it, especially the Knative community.
I've intentionally not put in examples because filters at this level have custom conformance.